src/components/studentSummary.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Row, Col, Container, Stack } from 'react-bootstrap'
+import SummaryBox from './summaryBox'
+
+function StudentSummary() {
+  return (
+    <>
+      <Stack direction="horizontal" style={{ justifyContent: 'space-between', margin:"20px" }}>
+        <div>
+          <div className="subHdg">Students Summary</div>
```

Comment:
css class names should be more meaningful. If it's related to some feature then it should be like {feature}-sub-heading